drivers/dai/intel/dmic/dmic.c

```diff
@@ -174,30 +250,55 @@ static void dai_dmic_irq_handler(const void *data)
 
 static inline void dai_dmic_dis_clk_gating(const struct dai_intel_dmic *dmic)
 {
+#if CONFIG_SOC_SERIES_INTEL_CAVS_V15
+	(void)index;
```

Comment:
where this variable comes from ?

Reply:
copy paste error from sof code... removing